### PR TITLE
pyHipify Fixes

### PR DESCRIPTION
**Pull request by rohithkrn**
This PR makes the following changes:
* stores cuda_to_hip mappings in python OrderedDicts
* Replace cudaError with cudaError_t and remove cudaError mapping 

@bddppq @petrex

### CODE REVIEW

**Comment by petrex** [comment]
thanks @rohithkrn

File: tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py [hunk 1]
```diff
@@ -12,2213 +14,2225 @@
...
+    ("std::floor", ("::floor")),
+    ("std::exp", ("::exp")),
+    ("std::log", ("::log")),
+    ("std::pow", ("::pow")),
+    ("std::fabs", ("::fabs")),
+    ("std::fmod", ("::fmod")),
+    ("std::remainder", ("::remainder")),
+]
+
+MATH_TRANSPILATIONS = collections.OrderedDict(MATH_TRANSPILATIONS)
```
Comment (bddppq) [inline tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py]: You can combine these two expressions into one:
```python
MATH_TRANSPILATIONS = collections.OrderedDict([
    ("std::max", ("::max")),
    ("std::min", ("::min")),
    ("std::ceil", ("::ceil")),
    ("std::floor", ("::floor")),
    ("std::exp", ("::exp")),
    ("std::log", ("::log")),
    ("std::pow", ("::pow")),
    ("std::fabs", ("::fabs")),
    ("std::fmod", ("::fmod")),
    ("std::remainder", ("::remainder")),
])
```

Comment (rohithkrn) [reply tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py]: Okay.

**Review by bddppq — APPROVED** [review]
LG Thanks

**Pull request closed without merging**